api-specs/friend.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package friendsapi;
+option go_package = "/friendsapi"
+
+import "google/protobuf/empty.proto";
+
+service FriendService {
+	rpc Add (FriendRequest) returns (google.protobuf.Empty);
+	rpc Accept (FriendRequest) returns (google.protobuf.Empty);
+	rpc Block (FriendRequest) return (google.protobuf.Empty);
+}
+
+message FriendRequest {
```

Comment:
do we need timestamp for date of request(?) to be able to display them by latest

Reply:
Good idea!